v1/src/main/java/com/google/cloud/teleport/spanner/ddl/InformationSchemaScanner.java

```diff
@@ -547,7 +538,9 @@ private void listIndexColumns(
         pkBuilder.end().endTable();
       } else {
         if (indexType != null && ordering != null) {
-          if ((indexType.equals("SEARCH") && !spannerType.equals("TOKENLIST"))
+          if ((indexType.equals("SEARCH")
```

Comment:
This is becoming a large boolean expression. Can you add a comment above the if block explaining the expression and why it is so complex? Can you simplify it if possible?